srv/salt/_modules/wait.py

```diff
@@ -144,6 +144,36 @@ def just(self):
         time.sleep(self.settings['delay'])
 
 
+class OSDInCheck(HealthCheck):
+    """
+    Check that all OSDs are "in"
+    """
+
+    def __init__(self, **kwargs):
+        kwargs.update({'nohealthcheck': True})
```

Comment:
Instead of overriding `__init__` maybe add `nohealthcheck` to the kwargs in srv/salt/ceph/wait/until/all_in/default.sls?

Reply:
I normally think of the default.sls as potentially user configurable.  This particular check has no `status` keyword to send because of its nature.  I kept the only keyword in the default.sls to `timeout`.